packages/plugin-messagebroker/src/MessageBrokerPlugin.ts

```
@@ -0,0 +1,198 @@
+    }
+    if (this.redisService) {
+      await this.redisService.close();
+      this.redisService = undefined;
+    }
+    this.transportSubscriptions.clear();
+    this.logger.info(`${this.name} cleanup complete.`);
+  }
+
+  // --- ITransport implementation ---
+
+  async listen(topicOrQueueName: string, handler?: MessageHandler): Promise<string | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)